pkg/vpcmodel/grouping.go

```diff
@@ -85,18 +90,44 @@ func (g *groupedEndpointsElems) Name() string {
 	return listEndpointElemStr(*g, EndpointElem.Name)
 }
 
+func (g *groupedEndpointsElems) Names() (string, []string) {
+	names := func(ep EndpointElem) string {
+		myName, _ := ep.Names()
+		return myName
+	}
+	namesToPrint := endpointElemToPrint(*g, names)
+	return strings.Join(namesToPrint, commaSepartor), namesToPrint // todo Haim: let me know if you want something shorter at the first index
+}
```

Comment:
yes, it should be 20 chars max

Reply:
Cool, but what should the short name be then? I've put a placeholder